test/util/cart_utils.py

```diff
@@ -278,10 +281,18 @@ def launch_test(self, cartobj, cmd, srv1=None, srv2=None):
     def launch_cmd_bg(self, cartobj, cmd):
         """ launches the given cmd in background """
 
+        self.print_cmd("\nCMD : %s\n" % cmd)
+
         cmd = shlex.split(cmd)
         rtn = subprocess.Popen(cmd)
 
         if rtn is None:
             return -1
 
         return rtn
+
+    def print_cmd(self, cmd):
```

Comment:
since function is used in other places as well besides cmd printing, perhaps rename to generic print or log
Also, am surprised pylint didnt complain about not having comments on this function -- it tends to be unhappy if there is undocumented function

Reply:
Done. Also added comments to the fn.